src/vivarium/framework/population.py

```diff
@@ -219,16 +219,27 @@ def _order_initializers(self) -> None:
         while len(unordered_initializers) != starting_length:
             starting_length = len(unordered_initializers)
             for _ in range(len(unordered_initializers)):
-                initializer, columns_created, columns_required = unordered_initializers.pop()
+                initializer, columns_created, columns_required = unordered_initializers.popleft()
```

Comment:
the first _pop_initializer must always be ordered first, because it makes tracked and requires nothing, and everything else requires tracked. So, I flipped how we iterate over the unordered_initializers so that the first element is actually grabbed first by default

Reply:
That's also worth writing a test around.